app/src/main/java/vn/edu/usth/minigh/ProfileActivity.java

```diff
@@ -33,16 +33,14 @@
 
 public class ProfileActivity extends BaseActivity {
     public ProfileActivity() {
-        super(R.layout.activity_profile);
+        super(R.layout.activity_profile_alt);
     }
 
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
       
-        TextView txt_toolbar = (TextView) findViewById(R.id.main_text_bar);
-        txt_toolbar.setText("Profile");
```

Comment:
You meant this one?

Reply:
Now I get another problem: The title shows even when it's expanded.